### ISSUE TITLE
Add description meta tag contents

### ISSUE THREAD

**Issue by machour** [yiisoft-contrib/yiiframework.com#273]
Currently the values are empty: `<meta name="twitter:description" content="" />` so the generated preview looks a bit empty:

![captureaaa](https://user-images.githubusercontent.com/304450/37841309-b8772fb4-2ebf-11e8-9cf4-72bbd94e4413.PNG)

Awesome work!

**Comment by samdark** [comment]
Want to do a pull request?

**Comment by machour** [comment]
Yeah why not. I'll see this week-end how to have a contextual description for each page.

**Comment by rugabarbo** [comment]
In the past, I solved a similar problem for yiiframework.ru: https://github.com/samdark/yiiframework-ru/commit/50a59423fc8068de64516b4f1af032b191c1faa0

It can be useful for PR.
